### PR TITLE
feat: Filter empty log lines from ScanCode

### PR DESCRIPTION
**Pull request by mmurto**
Signed-off-by: Mikko Murto <mikko.murto@doubleopen.org>

### CODE REVIEW

File: apps/scanner_worker/src/index.ts [hunk 1]
```diff
@@ -181,7 +181,10 @@ const start = (): void => {
         });
 
         childProcess.stderr.on("data", (data) => {
-            console.log(`[${pid}] process: ${data}`);
+            // Filter empty lines
+            if (data.toString().trim() !== "") {
```
Comment (sschuberth) [inline apps/scanner_worker/src/index.ts:185]: Nit: Would it make sense to check on `.length == 0` for the trimmed string instead?

Comment (mmurto) [reply apps/scanner_worker/src/index.ts:185]: It would probably have been an equally solid implementation.

Comment (sschuberth) [reply apps/scanner_worker/src/index.ts:185]: More explicitly, what I was after is: Does it make sense to micro-optimize for not constructing a temporary empty string instance? I'm asking this serious as I know little about the optimizations done by JavaScript runtimes.

Comment (mmurto) [reply apps/scanner_worker/src/index.ts:185]: No idea of the performance difference, IMO no sense to try and micro-optimize it if there are no issues. The logs will be coming from ScanCode, so any micro-optimizations we'd be able to do here would be a drop in the bucket.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**